### PR TITLE
[TIGERDATA] Adding tigerdata application playbook

### PR DESCRIPTION
**Pull request by carolyncole**
related to https://github.com/pulibrary/tiger-data-app/issues/8

### CODE REVIEW

File: group_vars/tigerdata/common.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+application_db_name: '{{ app_db_name }}'
+application_dbuser_name: '{{ app_db_user }}'
+application_dbuser_password: '{{ app_db_password }}'
+application_dbuser_role_attr_flags: 'CREATEDB'
+app_host_name: '{{ passenger_server_name }}'
+application_host_protocol: 'https'
+running_on_server: true
+tigerdata_honeybadger_key: '{{ vault_honeybadger_api_key }}'
+bundler_version: '2.3.26'
+desired_nodejs_version: 'v16.19.0'
```
Comment (mccalluc) [inline group_vars/tigerdata/common.yml:19]: tiger-data-app tool-versions has
```
nodejs 18.14.0
```
Would it be safer for these to match?

File: group_vars/tigerdata/production.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      init_config:
+      instances:
+        - server: lockers-and-study-spaces.princeton.edu
+          port: 443
+  - type: process
+    configuration:
+      init_config:
+      instances:
+        -   name: lockers-and-study-spaces
+            service: lockers-and-study-spaces
```
Comment (mccalluc) [inline group_vars/tigerdata/production.yml:34]: If this is a service that is used by multiple applications, maybe add a comment explaining that? As I'm reading this, my first guess is that it's a mistaken copy-paste, but I could be wrong.

File: group_vars/tigerdata/production.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        - nginx_status_url: http://localhost:81/nginx_status/
+      logs:
+        - type: file
+          path: /var/log/nginx/access.log
+          source: nginx
+        - type: file
+          path: /var/log/nginx/error.log
+          source: nginx
+        - type: file
+          path: /opt/lockers_and_study_spaces/current/log/production.log
```
Comment (mccalluc) [inline group_vars/tigerdata/production.yml:50]: ditto.

**Review by mccalluc — COMMENTED** [review]
A couple comments: if those lines are intentional, perhaps add a comment to explain.

File: group_vars/tigerdata/production.yml (hunk 2 above)
Comment (carolyncole) [reply group_vars/tigerdata/production.yml:34]: Yo are right this is a copy paste error.

File: group_vars/tigerdata/common.yml (hunk 1 above)
Comment (carolyncole) [reply group_vars/tigerdata/common.yml]: Unfortunately we do not have 18.14.0, but I updated it to18.13.0

File: inventory/all_projects/tigerdata [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[tigerdata_staging]
+tigerdata-staging1.princeton.edu
```
Comment (jrgriffiniii) [inline inventory/all_projects/tigerdata:2]: 👍

File: inventory/all_projects/tigerdata [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+tigerdata-staging2.princeton.edu
+[tigerdata_production]
+tigerdata-prod1.princeton.edu
```
Comment (jrgriffiniii) [inline inventory/all_projects/tigerdata:5]: 👍

**Review by jrgriffiniii — APPROVED** [review]

**Pull request merged**